### ISSUE TITLE
Installation error when attempting to install with ingest-attachment plugin

### ISSUE THREAD

**Issue by riverar** [elastic/windows-installers#309]
When attempting a first-time fresh install on a Windows Server 2012 R2 machine, installation fails. Upon log inspection, it appears this is related to `ingest-attachment` installation. While I cannot attach an entire log for security reasons, I've provided a sanitized log chunk and snippet below.

[snippet-install.log](https://github.com/elastic/windows-installers/files/3326810/snippet-install.log)

## Repro steps
1. Start installation of elasticsearch-7.2.0.msi.
2. On the Locations pivot, configure directories as needed.
3. On the Service pivot, Do not install a service.
4. On the Configuration pivot, select This is the first master in a new cluster.
5. On the Plugins pivot, select Ingest Attachment Processor and ICU Analysis.
6. Continue with defaults and observe installation failure.

## Log snippet
```
Elastic.InstallerHosts.Elasticsearch.Tasks.Install.InstallPluginsTask: Installing Elasticsearch plugins
[0] installing ingest-attachment installing ingest-attachment
-> Downloading ingest-attachment from elastic
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.bouncycastle.jcajce.provider.drbg.DRBG (file:/C:/Program%20Files/Elastic/Elasticsearch/7.2.0/lib/tools/plugin-cli/bcprov-jdk15on-1.61.jar) to constructor sun.security.provider.Sun()
WARNING: Please consider reporting this to the maintainers of org.bouncycastle.jcajce.provider.drbg.DRBG
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@     WARNING: plugin requires additional permissions     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
* java.lang.RuntimePermission accessClassInPackage.sun.java2d.cmm.kcms
* java.lang.RuntimePermission accessDeclaredMembers
* java.lang.RuntimePermission getClassLoader
* java.lang.reflect.ReflectPermission suppressAccessChecks
* java.security.SecurityPermission createAccessControlContext
* java.security.SecurityPermission insertProvider
* java.security.SecurityPermission putProviderProperty.BC
See http://docs.oracle.com/javase/8/docs/technotes/guides/security/permissions.html
for descriptions of what these permissions allow and the associated risks.
-> Installed ingest-attachment
[0] installing ingest-attachment installing ingest-attachment
System.Exception: Execution failed (C:\Program Files\Elastic\Elasticsearch\7.2.0\bin\elasticsearch-plugin.bat install ingest-attachment --batch). ExitCode: 0 ErrorCheck: True ErrorOut:WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.bouncycastle.jcajce.provider.drbg.DRBG (file:/C:/Program%20Files/Elastic/Elasticsearch/7.2.0/lib/tools/plugin-cli/bcprov-jdk15on-1.61.jar) to constructor sun.security.provider.Sun()
WARNING: Please consider reporting this to the maintainers of org.bouncycastle.jcajce.provider.drbg.DRBG
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by ygel** [comment]
#311

**Comment by riverar** [comment]
@ygel Just heads up, this issue is older than 311. 😄

**Comment by ygel** [comment]
Correct, it is very likely that it's fixed in v7.2.0, which has its own gremlins ...
